Controllers/HomeController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("")]
+		[Route("app")]
+		[Route("app/{path}")]
+		[Route("app/product/{productId}")]
+		public IActionResult App(int productId = 0, string path = null)

```

Comment:
Style: Use capital `S` `String`.

The reason for this is to get consistent syntax highlighting of class/struct names.  `String` in this case is interpreted as a class by the syntax highlighter, but `string` is interpreted as a keyword.  Also, all other classes start with a capital letter in all of these files, so having `String` be a special case leads to confusion.